### PR TITLE
ISSUE-67, update to tournament-v5

### LINKED ISSUE #67: Tournament V5
Are there plans to support Tournament v5?

### PR DESCRIPTION
**Pull request by happsie**
Resolves #67

### CODE REVIEW

File: riot/lol/constants.go [hunk 1]
```diff
@@ -38,14 +38,14 @@ const (
...
+	endpointTournamentStubBase                 = endpointBase + "/tournament-stub/v5"
 	endpointCreateStubTournamentCodes          = endpointTournamentStubBase + "/codes?count=%d&tournamentId=%d"
 	endpointGetStubLobbyEvents                 = endpointTournamentStubBase + "/lobby-events/by-code/%s"
 	endpointCreateStubTournamentProvider       = endpointTournamentStubBase + "/providers"
 	endpointCreateStubTournament               = endpointTournamentStubBase + "/tournaments"
-	endpointTournamentBase                     = endpointBase + "/tournament/v4"
+	endpointTournamentBase                     = endpointBase + "/tournament/v5"
 	endpointCreateTournamentCodes              = endpointTournamentBase + "/codes?count=%d&tournamentId=%d"
-	endpointGetLobbyEvents                     = endpointTournamentBase + "/lobby-events/by-code/%s"
+	endpointGetLobbyEvents                     = endpointTournamentBase + "/lobby-events/by-code/%s"
```
Comment (KnutZuidema) [inline riot/lol/constants.go:48]: nitpick, but could you please remove this change.

**Review by KnutZuidema — COMMENTED** [review]
Thanks for taking this up 👍🏼

File: riot/lol/constants.go (hunk 1 above)
Comment (happsie) [reply riot/lol/constants.go]: Sorry, fat fingers. Fixed it, but I will squash the commits as well. So you can wait to trigger a new build 😃

**Comment by happsie** [comment]
@KnutZuidema I have fixed the formatting issue, also moved to regional routing instead of platform routing for the tournament API. Missed that in the first draft. Feel free to trigger build if you feel like continuing with this change

**Comment by happsie** [comment]
@KnutZuidema I can't understand what's wrong with the lint. Do you?

**Comment by KnutZuidema** [comment]
@happsie they are broken, sadly. Thanks for this, LGTM 👍🏼

**Review by KnutZuidema — APPROVED** [review]

**Pull request merged**